learning_assistant/models.py

```diff
@@ -1,3 +1,39 @@
 """
 Database models for learning_assistant.
 """
+from django.db import models
+from model_utils.models import TimeStampedModel
+
+
+class CoursePrompt(TimeStampedModel):
+    """
+    This model represents a mapping between a particular course ID and a text prompt associated with the course ID.
+
+    .. no_pii: This model has no PII.
+    """
+
+    # course ID with which the text prompt is associated with
```

Comment:
Nit. You say "with" twice here.

```suggestion
    # course ID with which the text prompt is associated
```